specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/examples/KustoPoolDataConnectionsUpdate.json

```diff
@@ -18,6 +18,9 @@
   },
   "responses": {
     "200": {
+      "headers": {
```

Comment:
Can you please share the reason for this change?

Reply:
I will revert this change